src/types/Article.ts

```diff
@@ -0,0 +1,32 @@
+export interface Article {
+  title: string;
+  slug: string;
+  body: string;
+  createdAt: Date;
+  updatedAt: Date;
+  description: string;
+  tagList: [];
```

Comment:
What does this array contain? Is it an array of strings? If that's the case, then use `string[];`

Reply:
I don't know, because the tags don't work in the backend. Always returns empty `[]`